src/main/kotlin/car/domain/Winners.kt

```diff
@@ -0,0 +1,19 @@
+package car.domain
+
+class Winners(carMoveHistoryCollection: CarMoveHistoryCollection) {
+    val winners: List<Winner>
+
+    init {
+        val maxPosition = carMoveHistoryCollection.moveHistories
+            .map { it.position }
+            .max()!!
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
            .max() ?: 0
```